src/components/views/elements/StyledCheckbox.tsx

```diff
@@ -16,7 +16,7 @@ export enum CheckboxStyle {
 }
 
 interface IProps extends React.InputHTMLAttributes<HTMLInputElement> {
-    inputRef?: React.RefObject<HTMLInputElement>;
+    inputRef?: RefCallback<HTMLInputElement>;
```

Comment:
could this not be `Ref` to accept either?

Reply:
https://github.com/element-hq/element-web/pull/28452/commits/b8e54750a06ebd4bcbaf69236a2e8e9315565810